src/main/java/net/sf/jabref/gui/fieldeditors/FieldEditorFocusListener.java

```diff
@@ -23,7 +23,7 @@ public void focusGained(FocusEvent event) {
     @Override
     public void focusLost(FocusEvent event) {
         if (event.getSource() instanceof FieldEditor) {
-            ((FieldEditor) event.getSource()).setValidBackgroundColor();
+            //Do nothing
```

Comment:
I cannot comment if this is the correct way to implement the problem right now, but if it is please rephrase the condition to 
```java
if (!(event.getSource() instanceof FieldEditor)) {
...
}
```